### ISSUE TITLE
Paid Stats / Components: Pricing Slider A11y review

### ISSUE THREAD

**Issue by kangzj** [Automattic/jetpack#31478]
A round of A11y review

- Keyboard support
- Focus
- Screen reader support?

**Comment by dognose24** [comment]
> Keyboard support

The slider component has implemented the keyboard control for moving the thumb by the **_left and right arrows_** and **_page up & down_** keys.

**Comment by dognose24** [comment]
> Focus

The focus styling should be updated  with box shadow as shown in the video below:

https://github.com/Automattic/jetpack/assets/6869813/621c300b-1b07-427a-8e93-6e177ed44eda

I feel like the box shadow for the focused status has highlighted users' operations. However, I noticed the prototype shows a box shadow between **_dragging_** and **_dropping_** the thumb rather than the focused status, like the video below:

https://github.com/Automattic/jetpack/assets/6869813/9ecf122b-bcd5-4879-b7ac-7a04b2495caa

Both solutions look good to me. Which one do you refer to, @keoshi @ederrengifo? 🙂

**Comment by ederrengifo** [comment]
I like option 2. Showing a focus style only when there is an interaction seems more intuitive since it also alerts the user about the action that is happening, even when using the keyboard keys (I assume that's what is happening by the end of both videos). Thoughts? cc @keoshi

**Comment by keoshi** [comment]
Agree with @ederrengifo — having that extra visual styling only when interacting with the slider is delightful. It would be great to keep the element focused when changing the value using the keyboard (like option 1), but I don't mind that it's only temporary.

**Comment by dognose24** [comment]
The final version of the operation styling would look like the below:

https://github.com/Automattic/jetpack/assets/6869813/fdc8dee4-74ba-434b-8b4c-977616db0d54

**Comment by kangzj** [comment]
thanks for checking all those. I've marked it as finished. Great job! 👍
